whetstone-worker/src/main/java/com/deliveryhero/whetstone/worker/WhetstoneWorkerInitializer.kt

```diff
@@ -21,9 +23,11 @@ public class WhetstoneWorkerInitializer : Initializer<WorkManager> {
 
 
 private fun Whetstone.installWorkerFactory(application: Application) {
```

Comment:
random thought, non-blocking. is there merit to making this function a suspending function called in a coroutine scope? those are also non-blocking even when called on the main thread. cc @vsdeni